File: src/test/java/com/tom/StudentTest.java
package com.tom;

import com.tom.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestScoreTest(){
        Student student = new Student("Brous",95,72);
        Assertions.assertEquals(95,student.hightest());
    }

    @Test
    public void averageTest(){
        Student student = new Student("Bob",62,87);
        Assertions.assertEquals((62+87)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/tom/StudentTest.kt
package com.tom.kotlin

import com.tom.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun hightestTest(){
        val student = Student("Dora",45,72)
        Assertions.assertEquals(72,student.hightest())
    }

    @Test
    fun averageTest(){
        val student = Student("Dora",45,72)
        Assertions.assertEquals((45+72)/2,student.getAverage())
    }

    @Test
    fun gradingTest(){
        val student = Student("Dora",45,72)
        Assertions.assertEquals('F',student.grading())
    }

}